index.js

```diff
@@ -17,10 +17,9 @@ var net = require("net"),
 // can set this to true to enable for all connections
 exports.debug_mode = false;
 
-var arraySlice = Array.prototype.slice;
 function trace() {
     if (!exports.debug_mode) return;
-    console.log.apply(null, arraySlice.call(arguments));
+    console.log.apply(null, to_array.call(arguments));
```

Comment:
This part is actually obsolete too as I later realized and will be superseded by debug (or what ever it'll later be).